resources/language/resource.language.es_es/strings.po

```diff
@@ -838,7 +841,7 @@ msgstr "No hay un dispositivo que maneje las actualizaciones automáticas de la
 
 msgctxt "#30213"
 msgid "InputStream Helper settings..."
-msgstr "Ajustes de InputStream Helper..."
+msgstr "Configurar InputStream Helper..."
 
```

Comment:
DON'T AGREE for consistency with the rest of the translation (but both are valid).